commands/debug/root.go

```diff
@@ -0,0 +1,50 @@
+package debug
+
+import (
+	"github.com/docker/cli/cli/command"
+	"github.com/spf13/cobra"
+)
+
+// DebugConfig is a user-specified configuration for the debugger.
+type DebugConfig struct {
+
```

Comment:
nit: empty line (also below)

Reply:
Off-topic: is there any way we can get this linted / auto-formatted? I know that `gofumpt` has a rule for this, but there's some other rules in there that are a bit more controversial IMO. @crazy-max any ideas?

Reply:
Ah they have rules like this for functions at least in "No empty lines around function bodies"